api/datastore/sql/migrations/1_add_route_created_at.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"database/sql"
+	"github.com/pressly/goose"
+)
+
+// we skip the error here to make previous datastore tables work fine
+func up1(tx *sql.Tx) error {
+	_, err := tx.Exec("ALTER TABLE routes ADD created_at text;")
+	return checkMigrationsUpError(err)
```

Comment:
we shouldn't be doing this in any of our migrations, the migration should be written differently and not rely on this behavior

Reply:
this being 'checkMigrationsUpError' - delete this

Reply:
this was necessary in that case if we want to keep table definitions as they were (with new tables from the migrations), since tables where adjusted, there's no point to hide dup. column error.